### ISSUE TITLE
Parsing of NG number wrong

### ISSUE THREAD

**Issue by nicpottier** [nyaruka/phonenumbers#80]
See: https://play.golang.org/p/I76HGKRGPKG

Compare with upstream google which marks that number valid.

**Comment by vasundhara785** [comment]
You have to add "+" before the number to indicate international number.
It says valid if we add "+"

**Comment by nicpottier** [comment]
No, this should work for both. And it did. Looks like the changes in v1.0.60 broke this. I'm investigating.
